packages/ibm-products/src/components/EditTearsheet/EditTearsheetForm.tsx

```diff
@@ -22,6 +22,53 @@ const defaults = {
   hasFieldset: true,
 };
 
+interface EditTearsheetFormBaseProps {
+  /**
+   * Content that shows in the tearsheet form
+   */
+  children?: ReactNode;
+
+  /**
+   * Sets an optional className to be added to the tearsheet form
+   */
+  className?: string;
+
+  /**
+   * Sets an optional description on the form component
+   */
+  description?: string;
+
+  /**
+   * This optional prop will render your form content inside of a fieldset html element
+   * and is defaulted to true.
+   * You can set this prop to `false` if you have multiple fieldset elements or want to control the children of your Full Page's form content.
+   */
+  hasFieldset?: boolean;
+
+  /**
+   * Sets an optional subtitle on the form component
+   */
+  subtitle: string;
```

Comment:
This should be optional
```suggestion
  subtitle?: string;
```